app/Policies/MaterialPolicy.php

```diff
@@ -30,6 +30,11 @@ public function materialToggleAvailability(User $user,Material $material)
         return $user->id == $material->user_id;
     }
 
+    public function authCheck()
+    {
+        return backpack_auth()->check();
```

Comment:
what if we decide to allow non-logged users to see the app's materials?
the authorization responsibility falls only on policies